packages/opencensus-core/src/metrics/export/export-component.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {MetricProducerManager} from './metric-producer-manager';
+
+// Class that holds the implementation instance for MetricProducerManager.
+export class ExportComponent {
+  private static readonly METRIC_PRODUCER_MANAGER = new MetricProducerManager();
+
+  /**
+   * Returns the global MetricProducerManager which can be used to register
+   * handlers to export all the recorded metrics.
+   *
+   * @return {MetricProducerManager}.
+   */
+  getMetricProducerManager(): MetricProducerManager {
+    return ExportComponent.METRIC_PRODUCER_MANAGER;
```

Comment:
Is this just to keep the MetricProducerManager as a singleton? I'm not sure this needs to be a class at all if that's the case (vs simply exporting a const).

Reply:
SGTM, Done. PTAL